README.md

```diff
@@ -58,17 +58,13 @@ The easiest way to start playing is
  2. Copy OAI_CONFIG_LIST_sample to ./notebook folder, name to OAI_CONFIG_LIST, and set the correct configuration.
  3. Start playing with the notebooks!
 
-## Using existing docker image
-Install docker, save your oai key into an environment variable name OPENAI_API_KEY, and then run the following.
+## [Installation](https://microsoft.github.io/autogen/docs/Installation)
 
-```
-docker pull yuandongtian/autogen:latest
-docker run -it -e OPENAI_API_KEY=$OPENAI_API_KEY -p 8081:8081 docker.io/yuandongtian/autogen:latest
-```
+### Option 1. Install and Run AutoGen in Docker
 
-Then open `http://localhost:8081/` in your browser to use AutoGen. The UI is from `./samples/apps/autogen-assistant`. See docker hub [link](https://hub.docker.com/r/yuandongtian/autogen) for more details.
+Find detailed instructions for users [here](https://microsoft.github.io/autogen/docs/Installation#option-1-install-and-run-autogen-in-docker), and for developers [here](https://microsoft.github.io/autogen/docs/Contribute#docker).
 
-## Installation
+### Option 2. Install AutoGen Locally without Docker
```

Comment:
This can be confusing: One can install autogen and docker and use autogen out of docker but run code in docker. Check line 86.

Reply:
how about now?